docs/_docs/installation.md

```diff
@@ -1,138 +1,164 @@
 
-<div class="note info">
-  <h5>Problems installing Jekyll?</h5>
-  <p>
-    Check out the <a href="../troubleshooting/">troubleshooting</a> page or
-    <a href="{{ site.repository }}/issues/new">report an issue</a> so the
-    Jekyll community can improve the experience for everyone.
-  </p>
-</div>
+## Install on macOS {#macOS}
 
-<div class="note info">
-  <h5>Running Jekyll on Windows</h5>
-  <p>
-    While Windows is not officially supported, it is possible to get Jekyll running
-    on Windows. Special instructions can be found on our
-    <a href="../windows/#installation">Windows-specific docs page</a>.
-  </p>
-</div>
+We only cover latest macOS High Sierra 10.13 here, which comes with Ruby 2.3.3, older systems will need to [install a more recent Ruby version via Homebrew](#homebrew).
 
-## Install with RubyGems
+First, you need to [install Xcode from the App Store](https://itunes.apple.com/app/xcode/id497799835?mt=12) or [from Apple Developer website](https://developer.apple.com/xcode/). Then to install the command-line tools, open a terminal and run:
+
+```sh
+xcode-select --install
+```
 
-The best way to install Jekyll is via
-[RubyGems](https://rubygems.org/pages/download). At the terminal prompt,
-simply run the following command to install Jekyll:
+Check your Ruby version, and upgrade [RubyGems](https://rubygems.org/pages/download):
 
 ```sh
-gem install jekyll
+ruby -v
+2.3.3
+
+gem update --system
 ```
 
-All of Jekyll’s gem dependencies are automatically installed by the above
-command, so you won’t have to worry about them at all.
-
-<div class="note info">
-  <h5>Installing Xcode Command-Line Tools</h5>
-  <p>
-    If you run into issues installing Jekyll's dependencies which make use of
-    native extensions and are using macOS, you will need to install Xcode
-    and the Command-Line Tools it ships with. Download them in
-    <code>Preferences &#8594; Downloads &#8594; Components</code>.
-  </p>
-</div>
+Great, let's install Jekyll. We also need [bundler](https://bundler.io/) to help us handle [plugins](../plugins) and [themes](../themes):
 
-## Pre-releases
+```sh
+gem install bundler jekyll
+```
 
-In order to install a pre-release, make sure you have all the requirements
-installed properly and run:
+You're ready to go, either by installing our default jekyll blog theme with `jekyll new jekyll-website` or by starting from scratch:
 
 ```sh
-gem install jekyll --pre
+mkdir jekyll-website
+cd jekyll-website
+
+# Create a Gemfile
+bundle init
+
+# Add Jekyll
+bundle add jekyll
+
+# Install gems
+bundle install
 ```
 
-This will install the latest pre-release. If you want a particular pre-release,
-use the `-v` switch to indicate the version you'd like to install:
+From there you can either use a [theme](../themes/) or create your own layouts.
+
+### Install via Homebrew {#homebrew}
```

Comment:
If macOS 10.13 has the appropriate version requirements for Ruby out of the box, I'm not sure I see a reason to add a section on installing via Homebrew. 

If we want to keep this section. I think it would be better if we rearrange the documentation to be something along the lines of:

```
                 |-> Set up Ruby included with the OS  +
Install on macOS +                                     | -> Install Jekyll -> Generate your first website
                 |-> Install a newer Ruby via Homebrew +

```

I was quite confused when I got to this section for two reasons:
1. We don't install Jekyll via Homebrew, so just what the heck is going on here :smiley:
2. We already set up Ruby, we don't need to do it again.

What do y'all think?